rfc/9/index.md

```diff
@@ -0,0 +1,342 @@
+
+This RFC explicitly prohibits embedding a zipped OME-Zarr file as subhierarchy of a parent OME-Zarr hierarchy.
+In particular this prohibits "recursive zipping", the embedding of a zipped OME-Zarr file within a parent zipped OME-Zarr file.
+This restriction may be revised in the future, especially in the light of the "collections" RFC.
+
+Furthermore, this RFC prohibits splitting up the ZIP archive into multiple files ("multi-volume archives"), in favor of directory-backed OME-Zarr and Zarr's sharding codec.
+
+Finally, this RFC also defines a new file extension to be used specifically with zipped OME-Zarr files.
+This should enable file type detection (in absence of a magic number), improve user experience (e.g. by enabling file type association), avoid "accidental" in-place extraction (e.g. using on-board tooling of some operating systems) and encourage the use of OME-Zarr-specific tooling for creating zipped OME-Zarr files (to follow the recommendations listed earlier).
+
+Note that this RFC does not - semantically or otherwise - restrict the data content of OME-Zarr hierarchies to be stored in zipped OME-Zarr files.
+
+## Specification
+
+Amend the specification with the following section:
+
+### Single-file OME-Zarr
+
+This section specifies how to store an OME-Zarr hierarchy within a single file.
+
+#### Zipped OME-Zarr files
+
+An OME-Zarr hierarchy MAY be stored within a ZIP archive.
+
+For a ZIP file to be referred to as a single-file OME-Zarr, it MUST contain exactly one OME-Zarr hierarchy.
+The root of the ZIP archive MUST correspond to the root of the OME-Zarr hierarchy, and it MUST contain the the OME-Zarr's root-level `zarr.json`.
+
+When creating zipped OME-Zarr files, the following are RECOMMENDED:
+
+- The ZIP64 format extension SHOULD be used to allow for files larger than 4 GiB.
+- ZIP-level compression SHOULD be disabled in favor of Zarr-level compression codecs.
+- The sharding codec SHOULD be used to reduce the number of entries within the ZIP archive.
+- The root-level `zarr.json` file SHOULD be the first ZIP file entry and the first entry in the central directory header; other `zarr.json` files SHOULD follow immediately afterwards, in breadth-first order.
+
+These recommendations are REQUIRED for a zipped OME-Zarr file specialization from here on referred to as "recommended variant".
+
+If a zipped OME-Zarr file is of the recommended variant, its ZIP archive comment SHOULD contain null-terminated UTF-8-encoded JSON with an `ome` attribute that holds a `version` key with the OME-Zarr version as string value, equivalent to `{"ome": { "version": "XX.YY" }}`.
+Conversely, zipped OME-Zarr files that do not meet all requirements for the recommended variant SHALL NOT contain this archive comment.
+
+Zipped OME-Zarr files SHALL NOT be embedded in a parent OME-Zarr hierarchy (as a sub-hierarchy or otherwise).
```

Comment:
```suggestion
OME-Zarr zip files SHALL NOT be embedded in a parent OME-Zarr hierarchy (as a sub-hierarchy or otherwise).
```

Actually, I'm not sure about this paragraph. I do not want OME-Zarr zip files in other OME-Zarr zip files at this time, but I do not see a particular problem with these representing part of a hierarchy that is unzipped.